webapp/tests/karma/utils.js

```diff
@@ -24,14 +24,15 @@ window.KarmaUtils = {
   nullPromise: () => () => Q.defer().promise,
   mockDB: db => () => () => db,
   setupMockStore: function(initialState, mocks = {}) {
+    /* ToDo: migrate and/or clean
```

Comment:
This will be cleaned in this ticket: https://github.com/medic/angular10-migration/issues/126